test/test_format_az.py

```diff
@@ -0,0 +1,567 @@
+
+
+
+
+
+    def test_convert_scientific_notation(self):
+        self.assertEqual(pronounce_number(0, scientific=True), "sıfır")
+        self.assertEqual(pronounce_number(33, scientific=True),
+                         "üç nöqtə üç vurulsun on üstü bir")
+        self.assertEqual(pronounce_number(299792458, scientific=True),
+                         "iki nöqtə doqquz doqquz vurulsun on üstü səkkiz")
+        self.assertEqual(pronounce_number(299792458, places=6,
+                                          scientific=True),
+                         "iki nöqtə doqquz doqquz yeddi doqquz iki beş vurulsun "
+                         "on üstü səkkiz")
+        self.assertEqual(pronounce_number(1.672e-27, places=3,
+                                          scientific=True),
+                         "bir nöqtə altı yeddi iki vurulsun on üstü "
+                         "mənfi iyirmi yeddi")
+
+    def test_auto_scientific_notation(self):
+        self.assertEqual(
+            pronounce_number(1.1e-150), "bir nöqtə bir vurulsun "
+                                        "on üstü mənfi yüz əlli")
+        # value is platform dependent so better not use in tests?
+        # self.assertEqual(
+        #    pronounce_number(sys.float_info.min), "iki nöqtə iki iki times "
+        #                                          "on to the power of "
+        #                                          "mənfi üç yüz "
+        #                                          "və səkkiz")
+        # self.assertEqual(
+        #    pronounce_number(sys.float_info.max), "bir nöqtə yeddi doqquz "
+        #                                          "times on to the power of"
+        #                                          " üç yüz və səkkiz")
+
+
+    def test_large_numbers(self):
+        self.assertEqual(
+            pronounce_number(299792458, short_scale=True),
+            "iki yüz doxsan doqquz milyon, yeddi yüz "
+            "doxsan iki min, dörd yüz əlli səkkiz")
+        self.assertEqual(
+            pronounce_number(299792458, short_scale=False),
+            "iki yüz doxsan doqquz milyon, yeddi yüz "
+            "doxsan iki min, dörd yüz əlli səkkiz")
+        self.assertEqual(
+            pronounce_number(100034000000299792458, short_scale=True),
+            "yüz kvintilyon, otuz dörd kvadrilyon, "
+            "iki yüz doxsan doqquz milyon, yeddi yüz "
+            "doxsan iki min, dörd yüz əlli səkkiz")
+        self.assertEqual(
+            pronounce_number(100034000000299792458, short_scale=False),
+            "yüz trilyon, otuz dörd min milyard, "
+            "iki yüz doxsan doqquz milyon, yeddi yüz "
+            "doxsan iki min, dörd yüz əlli səkkiz")
+        self.assertEqual(
+            pronounce_number(10000000000, short_scale=True),
+            "on milyard")
+        self.assertEqual(
+            pronounce_number(1000000000000, short_scale=True),
+            "bir trilyon")
+        # TODO maybe beautify this
```

Comment:
Is this relevant to the AZ test or just cut and pasted from the other file?

Reply:
No, sorry I used english files as template and edited them, and seems i have forgotten to remove some of non relevant comments.